scripts/windows/msi-check.ps1

```diff
@@ -0,0 +1,71 @@
+param (
+    [int]$Port = 3306,
+    [string]$InstallDir = "C:\Program Files\MariaDB"
```

Comment:
My only suggestion here:

Since we can Pass in $InstallDir, let's also pass in $BuildDir.